hack/verify-gofmt.sh

```diff
@@ -8,8 +8,9 @@ set -o pipefail
 
 GO_VERSION=($(go version))
 
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4') ]]; then
-  echo "Unknown go version '${GO_VERSION}', skipping gofmt."
+# require go version to be 1.5.1 to pin `go fmt` version to 829cc34.
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.5.1') ]]; then
```

Comment:
at the moment, Jenkins is running on 1.4.x again... so we need to park this until jenkins is on 1.5.x again